src/components/TradingStatePanel/style.scss

```diff
@@ -1,6 +1,10 @@
 @import "../../variables.scss";
 
 .hfui-tradingstatepanel__wrapper {
+  .hfui-panel__inner > div {
+    height: 100%;
+  }
+
```

Comment:
Can we do selectors more specific, using classes?

Reply:
This div is generated by `OutsideClickHandler` component, as far as I know it does not support className props